packages/gestalt/src/Button.js

```diff
@@ -248,7 +247,7 @@ const ButtonWithForwardRef: React$AbstractComponent<
         onTouchStart={handleTouchStart}
         ref={innerRef}
         style={compressStyle}
-        type={type}
+        type={type} // eslint-disable-line react/button-has-type
```

Comment:
@AlbertCarreras let's remove this `eslint-disable-line` completely and explicitly pass in the type:

```
if (props.type === 'submit') {
  return <button type="submit" />
}
```

Reply:
DONE in Button fix.